modules/ROOT/pages/software-extensibility.adoc

```diff
@@ -57,24 +54,18 @@ In the Bonita subscription edition, Bonita Studio contains xref:rest-api-extensi
 
 === Custom pages
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
-
 A xref:pages.adoc[custom page] is a page that you can add into Bonita Portal to form part of an application, customize information provided in default portal pages, or to add new features to the portal.
 To implement a page, you need to provide HTML, CSS and Javascript resources respecting some packaging constraints.
 You can create a page using the xref:ui-designer-overview.adoc[UI Designer], which automatically creates a well-formed page.
 Note that although the page framework provided in the product is stable, we cannot guarantee that all custom pages will work with future versions, because this depends on the details of how the page is implemented.
 
 === Custom widgets
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
-
 If the standard UI Designer xref:widgets.adoc[widgets] do not meet your needs, you can create a xref:custom-widgets.adoc[custom widget].
-You can then use your custom widget in pages, forms, and (for Subscription editions) fragments.
+You can then use your custom widget in pages, forms, and fragments - for Subscription editions.
```

Comment:
❓ Aren't fragments available in the Community edition in version 2021.1? or is it only available starting from a version later?

Reply:
ping